hack/test-pre-release-ossm-images.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+command -v jq >/dev/null 2>&1 || { echo "jq not found"; exit 1; }
+[[ -n "${TOKEN}" ]] || { echo "\$TOKEN is not set"; exit 2; }
+
+CLUSTER_CONTEXT=$(oc config current-context)
+KONFLUX_SERVER="api.stone-prod-p02.hjvn.p1.openshiftapps.com"
+
+echo "Login to konflux cluster to access the index image"
+oc login --token=$TOKEN --server=https://${KONFLUX_SERVER}:6443
+
+echo "Get the index image"
+INDEX_IMAGE=$(oc get releases -l appstudio.openshift.io/application=ossm-fbc-next --sort-by=.metadata.creationTimestamp -o name -n service-mesh-tenant | tail -1 | xargs oc -n service-mesh-tenant get -ojson | jq -r '.status.artifacts.index_image.index_image')
+INDEX_IMAGE=${INDEX_IMAGE##*/}
+
+echo "Switch back to kube cluster"
+oc config use-context $CLUSTER_CONTEXT
```

Comment:
```suggestion
oc config use-context "$CLUSTER_CONTEXT"
```
shellcheck suggests to use "Double quote to prevent globbing and word splitting"